unlock-app/package.json

```diff
@@ -31,6 +31,7 @@
     "history": "^4.7.2",
     "jest": "^23.6.0",
     "jest-styled-components": "^6.3.1",
+    "jsonwebtoken": "^8.4.0",
```

Comment:
HA! could you add a seperate PR just to add that dependency in `unlock-app`? This will make reverting things much easier should anything go wrong.

Reply:
No problem, just put a PR out for this 👍